rails_app/app/jobs/deals_scrape_job.rb

```diff
@@ -0,0 +1,20 @@
+class DealsScrapeJob < ApplicationJob
+  queue_as :default
+
+  def perform(*args)
+    # Do something later
+    deals_to_parse.each do |deal|
+      # new_deal_status = Settings.parser_helpers.join_parsers[deal.provider].constantize.new(deal.url)
+      new_deal = BasicParser.new(deal.url).parse
```

Comment:
This is definitely not true, parser type should be determined by something. `BasicParser` is just one option for that